packages/suite-desktop-core/e2e/tests/onboarding/t1b1/t1b1-recovery-success.test.ts

```diff
@@ -1,31 +1,7 @@
 import { test, expect } from '../../../support/fixtures';
 
-const mnemonic = [
-    'nasty',
-    'answer',
-    'gentle',
-    'inform',
-    'unaware',
-    'abandon',
-    'regret',
-    'supreme',
-    'dragon',
-    'gravity',
-    'behind',
-    'lava',
-    'dose',
-    'pilot',
-    'garden',
-    'into',
-    'dynamic',
-    'outer',
-    'hard',
-    'speed',
-    'luxury',
-    'run',
-    'truly',
-    'armed',
-];
+const mnemonic =
+    'nasty answer gentle inform unaware abandon regret supreme dragon gravity behind lava dose pilot garden into dynamic outer hard speed luxury run truly armed';
```

Comment:
it is less row hungry and I have changed few methods to expect string and transaform it to array on they own.
This way we can pass the mnemonic to emulator and our methods in same way (string) and not have to do splits or joins